### PR TITLE
Remove unnecessary <Spinner> and fix layout shift on media panel.

### LINKED ISSUE #724: Layout shift on Media panel
## Bug Description

ezgif com-video-to-gif (34)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by obetomuniz**
Fixed #724.

Kapture 2020-03-26 at 16 21 12

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/media/mediaPane.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ import styled from 'styled-components';
 /**
  * WordPress dependencies
  */
-import { Spinner } from '@wordpress/components';
```
Comment (obetomuniz) [inline assets/src/edit-story/components/library/panes/media/mediaPane.js:26]: It's interesting to note that this <Spinner /> component from Gutenberg is not displaying anything across the plugin. We are also using this component [here](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/header/buttons.js#L159), should we remove it there as well? Or maybe replace for something more useful?

**Review by barklund — APPROVED** [review]
Given that the spinner didn't really show anything, just take up space, this is an improvement. We might want to add some loader at some point though, but in an appropriate place and preferably something that actually works...

**Review by pbakaus — APPROVED** [review]

**Comment by pbakaus** [comment]
@barklund agree. Currently the "Loading.." in the media gallery kinda works, but it's not so great that it conflates two things:

1. a loading indicator
2. a measuring div for the intersection observer

In my branch that I dismissed, I started refactoring that a little, but dropped that work. /cc @spacedmonkey

File: assets/src/edit-story/components/library/panes/media/mediaPane.js (hunk 1 above)
Comment (swissspidy) [reply assets/src/edit-story/components/library/panes/media/mediaPane.js:26]: See #790 for that.

It's very odd that nothing is being displayed, didn't realize that. This definitely wasn't always the case. Probably due to some update when they changed how the components render, and now there is no stylesheet being loaded for it.

Even more of a reason to get rid of this.

**Comment by swissspidy** [comment]
Added to the alpha branch 👍

**Pull request merged**